R/call.R

```diff
@@ -919,6 +919,13 @@ call_parse_type <- function(call) {
   .Call(rlang_which_operator, call)
 }
```

Comment:
It might actually be more useful to generalize them to `r_add_would_overflow(intmax_t x, intmax_t y)`. i.e. don't assume that they should only be used with `r_ssize`

Reply:
Agreed that would be useful.